liquidator/OneInchExchange.js

```diff
@@ -0,0 +1,135 @@
+const Web3 = require("web3");
+
+const erc20Abi = require("./abi/IERC20.json");
```

Comment:
If this script is to follow the same pattern as others, this ABI should be passed in as a constructor param from the parent. See how the liquidator and disputer do this for reference.

Reply:
Question: Why not use `web3.eth.getGasPrice()`?